parcel.go

```diff
@@ -13,48 +15,133 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
 
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	var count int
+	err := s.db.QueryRow("SELECT COUNT (*) FROM parcel").Scan(&count)
+	if err != nil {
+		return 0, err
+	}
+	res, err := s.db.Exec("INSERT INTO parcel (number, client, status, address, created_at) VALUES ( :number, :client, :status, :address, :createdAt)",
```

Comment:
Можно так же не менять название переменной

```suggestion
	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES ( :number, :client, :status, :address, :created_at)",
```